.github/workflows/release.yml

```diff
@@ -69,35 +63,59 @@ jobs:
           aws s3 sync ./build "$BUCKET_PATH"
         working-directory: ./docs
 
-      - name: Push kolena, kolena-client dists to Test PyPI
+      - name: Push 'kolena' dist to Test PyPI
         uses: pypa/gh-action-pypi-publish@release/v1
         with:
           password: ${{ secrets.TEST_PYPI_API_TOKEN }}
           repository-url: https://test.pypi.org/legacy/
 
-      - name: Push kolena, kolena-client dists to PyPI
+      - name: Push 'kolena' dist to PyPI
         uses: pypa/gh-action-pypi-publish@release/v1
         with:
           password: ${{ secrets.PYPI_API_TOKEN }}
 
-      - name: Install twine for package distribution on CodeArtifact
-        run: pip3 install twine
+      - name: Create GitHub release with 'kolena' build as artifact
+        uses: marvinpinto/action-automatic-releases@v1.2.1
+        with:
+          repo_token: ${{ secrets.RELEASE_TOKEN }}
+          prerelease: false
+          files: |
+            ./dist/*.tar.gz
+
+      #
+      # backcompat: build and publish 'kolena-client' package
+      #
+
+      - name: Build 'kolena-client' package for backward compatibility
+        run: |
+          rm -rf ./dist
+
+          # update first instance of 'kolena' to 'kolena-client' in pyproject.toml, kolena/__init__.py (package name)
+          sed -i '0,/kolena/{s/kolena/kolena-client/}' pyproject.toml kolena/__init__.py
+
+          poetry install
```

Comment:
Should this be run again, or does it need to? It seems I can just run `poetry build` without additional commands?

Reply:
Updating the package name causes `poetry` to use a new environment, as environments are keyed on package name. Had a fun time debugging this one, which was [manifesting as `Command not found: sphinx-apidoc` in the following job step](https://github.com/kolenaIO/kolena/actions/runs/4995591035/jobs/8947648228).